packages/ERTP/src/amountMath.js

```diff
@@ -83,6 +83,10 @@ function makeAmountMath(brand, mathHelpersName) {
         return allegedAmount;
       }
       const { brand: allegedBrand, extent } = allegedAmount;
+      assert(
+        allegedBrand !== null,
```

Comment:
This should be `undefined` and not null. Can we add a test to hit this case to ensure it's catching the right thing?